content/get-started/install-brew.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Install via Homebrew
+seo-title: Install Iron Fish - Homebrew
+---
+
+> **Note:** Our Homebrew tap does not yet support **Apple Silicon/arm64**. We recommend installing Iron Fish [using NPM](/use/get-started/installation#install-or-update-iron-fish-win-mac). You can also install Iron Fish [from the source](#from-source) or [use Docker](#using-docker) with `x86_64` support.
```

Comment:
is this something I can help with? not sure what the priority is

Reply:
We would need to basically download different binaries based on the architecture. Brew supports this as bottles, https://docs.brew.sh/Bottles